tasks/run-matlab-tests/v0/task.json

```diff
@@ -21,6 +21,30 @@
             "required": false,
             "helpMarkDown": "Path to write code coverage report in Cobertura XML format."
         },
+        {
+            "name": "coberturaModelCoverage",
+            "type": "string",
+            "label": "Cobertura model coverage",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Path to write model coverage report in Cobertura XML format."
+        },
+        {
+            "name": "simulinkTestResults",
+            "type": "string",
+            "label": "Simulink test results",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Path to write Simulink test results in mldatx format."
```

Comment:
Path to export Simulink Test Manager results in MLDATX format.

Reply:
Done.